includes/class-wc-payments.php

```diff
@@ -296,8 +290,6 @@ public static function init() {
 			self::$card_gateway = new $card_class( self::$api_client, self::$account, self::$customer_service, self::$token_service, self::$action_scheduler_service, self::$failed_transaction_rate_limiter, self::$order_service );
 		}
 
-		self::maybe_register_platform_checkout_hooks();
```

Comment:
This code used to load in the `plugins_loaded` hook. Now, it will be loaded after the cart content is populated, inside the `init_platform_checkout()` function along with other platform checkout things.

Reply:
I'm worried moving it till after the 'woocommerce_cart_loaded_from_session' will cause problem because I think some of those hooks need to be registered sooner. It may work for you now but things could change or just happen in different orders on different sites.

Reply:
Good point. Instead of loading the entire `maybe_register_platform_checkout_hooks` in the  `woocommerce_cart_loaded_from_session` hook, maybe load only the `woocommerce_checkout_billing` action and `woocommerce_form_field_email` filter after cart is loaded.

This way, only the action/filter related to the email field is pushed off till the cart items are loaded. The rest of the platform checkout cookie script can stay where it is.

Reply:
Updated the code, it should be better. Let me know what you think!

Reply:
@superdav42 bump